### ISSUE TITLE
controls are disappearing?

### ISSUE THREAD

**Issue by thewonderfulsloth** [ControlCompany/ControlCompany#144]
as stated the controls on the top right disappear

it always happens after 2 or so days but i have no idea how to make it happen

1.12

its only happened recently aswell

**Comment by ShadowxSight** [comment]
For me the UI no longer appears when entering experimental mode with "0" but still functions I just can't tell what I am turning into.

**Comment by JustCallMeQuincy** [comment]
Do you have any other mods installed?

**Comment by ShadowxSight** [comment]
> Do you have any other mods installed?

Yes, however I can't be sure if they are the cause. I launched a solo just to check but never encountered the issue. My guess would be having your PC controlled alive and outside while spectating/spawning inside that causes it. But I have yet to get my friend or anyone to aid in testing to make sure that was the issue as I was always on surface when spawning on my friend inside when experiencing the UI bug.

**Comment by JawsyBoi** [comment]
I am having the same issue. I think it is because I added the LC_Office mod, which added 2 more mods (LethalNetworkAPI and Odin Serializer. The first page of the Experimental menu doesn't show, but the second page does.

**Comment by JawsyBoi** [comment]
And when I spawn in as that Shrimp monster, it stays on that when I get out of ghost mode

**Comment by ShadowxSight** [comment]
yes it is exactly LC_Office mod, I forgot me and my friend disabled it recently, but when mentioned by JawsyBoi I tried again solo with it enabled and encountered the same issue, while disabled no issue at all.

**Comment by klampzlamps** [comment]
LC mod compatibility has been added in v1.1.5, please update the mod and try again
